src/wp-includes/block-supports/duotone.php

```diff
@@ -32,6 +32,40 @@
  * @since 5.8.0
  */
 
+// Register the block support.
+WP_Block_Supports::get_instance()->register(
+	'duotone',
+	array(
+		'register_attribute' => array( 'WP_Duotone', 'register_duotone_support' ),
+	)
+);
+
```

Comment:
This block ought to be in the default-filters file.

Reply:
All of the other block supports (layout and position) have their hooks co-located with the support registration in block-supports. Should those be moved too? Or is it okay having these here?

Reply:
@ajlende let's leave it as is for now (including in this PR). They probably all should be in default-filters but probably a discussion for another time.